encrypted-media/content/content-metadata.js

```diff
@@ -10,6 +10,24 @@ content = addMemberListToObject( {
                     },
 
+    'mp4-securestop' : {
+        initDataType : 'cenc',
+        audio : {
+            type : 'audio/mp4;codecs="mp4a.40.2"',
+            path : '/encrypted-media/content/audio_aac-lc_128k_securestop.mp4'
+        },
+        video : {
+            type : 'video/mp4;codecs="avc1.4d401e"',
+            path : '/encrypted-media/content/video_512x288_h264-360k_enc_securestop.mp4'
+        },
+        keys : [{
+            kid : [0x10, 0x00, 0x00, 0x00, 0x10, 0x01, 0x10, 0x01, 0x10, 0x01, 0x10, 0x00, 0x00, 0x00, 0x00, 0x00],
+            key : [0xb7, 0xc8, 0xa9, 0x8f, 0x85, 0x05, 0x9c, 0x35, 0xf5, 0x1e, 0x95, 0x29, 0x45, 0xa5, 0x8e, 0xb2],
+            initDataType : 'cenc',
+            initData : 'AAADKnBzc2gAAAAAmgTweZhAQoarkuZb4IhflQAAAwoKAwAAAQABAAADPABXAFIATQBIAEUAQQBEAEUAUgAgAHgAbQBsAG4AcwA9ACIAaAB0AHQAcAA6AC8ALwBzAGMAaABlAG0AYQBzAC4AbQBpAGMAcgBvAHMAbwBmAHQALgBjAG8AbQAvAEQAUgBNAC8AMgAwADAANwAvADAAMwAvAFAAbABhAHkAUgBlAGEAZAB5AEgAZQBhAGQAZQByACIAIAB2AGUAcgBzAGkAbwBuAD0AIgA0AC4AMAAuADAALgAwACIAPgA8AEQAQQBUAEEAPgA8AFAAUgBPAFQARQBDAFQASQBOAEYATwA+ADwASwBFAFkATABFAE4APgAxADYAPAAvAEsARQBZAEwARQBOAD4APABBAEwARwBJAEQAPgBBAEUAUwBDAFQAUgA8AC8AQQBMAEcASQBEAD4APAAvAFAAUgBPAFQARQBDAFQASQBOAEYATwA+ADwASwBJAEQAPgBBAEEAQQBBAEUAQQBFAFEAQQBSAEEAUQBBAFIAQQBBAEEAQQBBAEEAQQBBAD0APQA8AC8ASwBJAEQAPgA8AEMASABFAEMASwBTAFUATQA+AFYATABHAEIAaQB2AE8ATwBsAEgAZwA9ADwALwBDAEgARQBDAEsAUwBVAE0APgA8AEwAQQBfAFUAUgBMAD4AaAB0AHQAcAA6AC8ALwBwAGwAYQB5AHIAZQBhAGQAeQAtAHQAZQBzAHQAcwBlAHIAdgBlAHIALgBhAHoAdQByAGUAdwBlAGIAcwBpAHQAZQBzAC4AbgBlAHQALwByAGkAZwBoAHQAcwBtAGEAbgBhAGcAZQByAC4AYQBzAG0AeAA/AFMAZQBjAHUAcgBlAFMAdABvAHAAPQAxACYAYQBtAHAAOwBVAHMAZQBTAGkAbQBwAGwAZQBOAG8AbgBQAGUAcgBzAGkAcwB0AGUAbgB0AEwAaQBjAGUAbgBzAGUAPQAxADwALwBMAEEAXwBVAFIATAA+ADwALwBEAEEAVABBAD4APAAvAFcAUgBNAEgARQBBAEQARQBSAD4A'
+        }]
+    },
+
```

Comment:
This new content is not used by any of the new tests, so it should not be in this PR.